eslint-plugin-kuc-v1/validator-in-update.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  create: function(context) {
+    return {
+      ClassDeclaration: function(node) {
+        const superClass = node.superClass.name;
+        if (superClass !== "KucBase") return;
+
+        const nodeBody = node.body;
+        if (nodeBody.type !== "ClassBody") return;
+
+        const sourceCode = context.getSourceCode();
+        const pattern = "validate[a-zA-z]+\\(";
```

Comment:
It seems there is an unexpected character on the pattern.
https://jex.im/regulex/#!flags=g&re=validate%5Ba-zA-z%5D%2B%5C%5C(

Reply:
```\\``` is used to escape ```\``` in JS.
You should change ```\\``` to ```\``` if you directly modify parameter "re" of https://jex.im/regulex/#!flags=g&re=
And you can directly copy the pattern string to the input box of https://jex.im/regulex, it can be escaped automatically.

Reply:
I got it. Thanks for the explanation.